lib/reek/cli/options.rb

```diff
@@ -9,7 +9,7 @@ module CLI
     #
     # See {file:docs/Command-Line-Options.md} for details.
     #
-    # :reek:TooManyInstanceVariables: { max_instance_variables: 9 }
+    # :reek:TooManyInstanceVariables: { max_instance_variables: 10 }
```

Comment:
Ah yes, I see. This needs some thinking on how this kind of problem should be handled in general.